internal/snmp/config.go

```diff
@@ -13,7 +13,7 @@ type ClientConfig struct {
 	// Path to mib files
 	Path []string `toml:"path"`
 	// Translator implementation
-	Translator string `toml:"translator"`
+	Translator string `toml:"-"`
```

Comment:
Not needed to be exported.
```suggestion
	translatorType string
```